space.go

```diff
@@ -194,6 +196,7 @@ func ConvertSpaces(request *goa.RequestData, spaces []*space.Space, additional .
 // ConvertSpace converts between internal and external REST representation
 func ConvertSpace(request *goa.RequestData, p *space.Space, additional ...SpaceConvertFunc) *app.Space {
 	selfURL := AbsoluteURL(request, app.SpaceHref(p.ID))
+	relatedIterationList := AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/iterations", p.ID.String()))
```

Comment:
We need to get a patch to Goa to generate Hrefs for other endpoints then just the top level Display resources.